### PR TITLE
Final About Page Work - Completed

### PR DESCRIPTION
**Pull request by MGS31**
NPM I for any missing components though they should all be there.

Final revisions made to the about page should look good at least from my perspective. Let me know if you want to see more changes.
Screenshot 2022-08-12 at 3 20 46 PM

### CODE REVIEW

**Review by b1u3too — APPROVED** [review]
Good to go! My only wishlist would be updating gitignore before merging, but we might want to do that together on main.

File: leafsmart-front-end/src/components/About.js [hunk 1]
```diff
@@ -1,17 +1,84 @@
...
+              className="grid justify-items-start bg-White rounded-full h-auto w-32 mx-auto mt-1"
+              src="/LeafSmartDark.png"
+              alt="Lgo"
+            />
+            <h1 className="text-8xl">LeafSmart</h1>
+            <p className="text-normal italic mb-2">
+              The place to start thinking of travelling!
+            </p>
+            <p>
+              LeafSmart is a independantly built app focusing on the
```
Comment (rachelpr) [inline leafsmart-front-end/src/components/About.js:24]: Just have a less wordy suggestion:
LeafSmart focuses on providing city data for our users' convenience. With technology at their fingertips, our users can find all they need to know about the locations they aim to travel to.

Comment (rachelpr) [reply leafsmart-front-end/src/components/About.js:24]: also means don't need to /br potentially

Comment (MGS31) [reply leafsmart-front-end/src/components/About.js]: Updated this blurb per your request!

**Review by rachelpr — APPROVED** [review]
🚀

**Pull request merged**